### PR TITLE
[stable/selenium] Make selenium nodes deployable as daemonset

### PR DESCRIPTION
**Pull request by staust**
#### Is this a new chart
no

#### What this PR does / why we need it:
This PR introduces the possibility to deploy chrome and firefox nodes as DaemonSet instead of Deployment. This is useful for (auto)scaling the cluster capabilities by only scaling node deployments.

Exact use case: 
We use the selenium cluster for high variable workloads of UI-tests. Therefore we want to scale the cluster on demand via auto scaling. As you could use specific node deployments for the chrome and firefox nodes via node selectors and anti affinities, it would be much easier to scale the selenium cluster by just scaling the corresponding node deployment and bypass the replication count (de-)increase by usage of a DaemonSet.

#### Which issue this PR fixes
no issue was created
#### Special notes for your reviewer:
I tested this via `helm template . -f values.yaml` with the different value combinations and applied the result via `kubectl`
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/selenium/templates/chrome-daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
+{{- if and (eq true .Values.chrome.enabled) (eq true .Values.chrome.runAsDaemonSet) -}}
+apiVersion: {{ template "deployment.apiVersion" . }}
+kind: DaemonSet
```
Comment (staust) [inline stable/selenium/templates/chrome-daemonset.yaml:3]: this is just a duplicate of the deployment file without the replication count

**Comment by staust** [comment]
@diemol do you have a rough estimation when you could have a look at these PR?
Is there anything I can support you with?

**Comment by staust** [comment]
@flah00 @ekirmayer you might want to look at this.

Sorry to be a bit annoying here

**Comment by staust** [comment]
> Please update the PR to resolve the conflicts, and then we can have a look.

Thx, did so.

**Comment by staust** [comment]
@diemol could you have a look again please?

**Review by diemol — APPROVED** [review]
Thank you @staust

**Pull request merged**